flowzone.yml

```diff
@@ -1533,6 +1534,32 @@ jobs:
           # Publish anything added to with the gh-release name
           files: ${{ runner.temp }}/gh-release-${{ github.event.pull_request.head.sha || github.event.head_commit.id }}/*
 
+      - name: Download source artifact
+        if: steps.gh_artifacts.outputs.count != '0'
+        uses: actions/download-artifact@v3
+        with:
+          name: source-${{ github.event.pull_request.head.sha || github.event.head_commit.id }}
+          path: ${{ runner.temp }}
```

Comment:
Tried to use the syntax `<<: *downloadSourceArtifact` so I could just override with the `if`, but GH didn't seem to like it